<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addcompany</class>
 <widget class="QMainWindow" name="addcompany">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
background-color: rgb(192, 221, 190);
}
QPushButton#pushButton_select_photo{
background-color: black;
color: white;
font: bold 15px;
border-radius: 10px;
transition: 0.2s;
}
QPushButton#pushButton_select_photo:hover {
background-color: rgb(80,80,80);
font: bold 17px;
}
QPushButton#pushButton_select_photo:pressed {
background-color: rgb(50,50,50);
}

QPushButton#pushButton_select_photo:focus{
box-shadow: 0 0 5px rgba(0,0,255,0.5);
font: bold 15px;
Width: 150px;
}
QPushButton#pushButton_back{
background-color: black;
color: white;
font: bold 15px;
border-radius: 10px;
transition: 0.2s;
}
QPushButton#pushButton_back:hover {
background-color: rgb(80,80,80);
font: bold 17px;
}
QPushButton#pushButton_back:pressed {
background-color: rgb(50,50,50);
}

QPushButton#pushButton_back:focus{
box-shadow: 0 0 5px rgba(0,0,255,0.5);
font: bold 15px;
Width: 150px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>918</width>
      <height>330</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/pishvazpic.JPG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name_company">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 15px;
border-radius: 8px;
border: 1px solid #0640ff;
padding: 5px;
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Enter companey name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_username">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 66, 51);</string>
    </property>
    <property name="text">
     <string>Company name :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_username_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 66, 51);</string>
    </property>
    <property name="text">
     <string>Address :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_username_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 66, 51);</string>
    </property>
    <property name="text">
     <string>Phone number :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_username_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 66, 51);</string>
    </property>
    <property name="text">
     <string>Email :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_phone">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 15px;
border-radius: 8px;
border: 1px solid #0640ff;
padding: 5px;
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="placeholderText">
     <string>Enter phone number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_email">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 15px;
border-radius: 8px;
border: 1px solid #0640ff;
padding: 5px;
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="placeholderText">
     <string>Enter email company</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_address">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 15px;
border-radius: 8px;
border: 1px solid #0640ff;
padding: 5px;
</string>
    </property>
    <property name="placeholderText">
     <string>Please enter your address</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>140</y>
      <width>181</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
border: 1px solid #0640ff;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_select_photo">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>310</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
border-top-left-radius: 30px;
border-bottom-right-radius: 30px;</string>
    </property>
    <property name="text">
     <string>select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>450</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
border-top-left-radius: 40px;
border-bottom-right-radius: 40px;</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Magneto</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(54, 66, 51);
color: white;
padding: 5px;
border-radius: 10px;
border-top-left-radius: 40px;
border-bottom-right-radius: 40px;

</string>
    </property>
    <property name="text">
     <string>PORJOB</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>918</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
